modules/tinymce/src/core/test/ts/browser/EditorRemoveTest.ts

```diff
@@ -29,6 +34,51 @@ UnitTest.asynctest('browser.tinymce.core.EditorRemoveTest', (success, failure) =
       cRemoveEditor
     ])),
 
+    Logger.t('remove editor after stylesheets load', Chain.asStep({}, [
+      Chain.op(() => {
+        // setup monitoring for uncaught errors
+        uncaughtErrors = [];
+        window.addEventListener('unhandledrejection', recordError);
+      }),
+      Chain.control(
+        McEditor.cFromHtml('<textarea></textarea>', {
+          ...settings,
+          setup: (editor: Editor) => {
+            editor.on('LoadContent', () => {
+              // Hook the function called when stylesheets are loaded
+              // so we can destroy the editor right after starting to load them.
+              const realLoadAll = editor.ui.styleSheetLoader.loadAll;
+              editor.ui.styleSheetLoader.loadAll = (...args) => {
+                realLoadAll.apply(editor.ui.styleSheetLoader, args);
+                editor.destroy();
+              };
+            });
+          }
+        }), (f, value, next, die, logs) => {
+          f(value, (value2, logs2) => {
+            die('Expected editor would not load completely', logs2);
+          }, (err, logs2) => {
+            // As we have deliberately removed the editor during the loading process
+            // we have to intercept the error that is thrown by McEditor.cFromHtml.
+            if (err === McEditor.errorMessageEditorRemoved) {
```

Comment:
I'm not too keen on doing the comparison this way, but I can't think of anything better off the top of my head.

Reply:
Well I had it just as a string originally but I didn't like that because if we ever changed it fixing all the usages would be a pain.

I was tempted to name it `ERROR_MESSAGE_EDITOR_REMOVED` ala Java style but I haven't seen us do that anywhere else so I avoided the temptation.

For now barring a better suggestion I'm going to mark this one resolved.